Readme.md

```diff
@@ -27,7 +27,8 @@ bevy_quickmenu = "0.1.5"
 ### Version Compatibility
 
 | Bevy Version | Crates Version |
-|--------------|----------------|
+| ------------ | -------------- |
+| 0.11.0       | 0.3.0          |
```

Comment:
This should be 0.12.0 for crate 0.3.0 (or even 0.12.1)

Reply:
Thanks for the feedback. It's been awhile since I've looked through my GH notifications--sorry for the delayed response.

This looks like it got merged in #10, so I'll close this PR.